dirsrvtests/tests/suites/fourwaymmr/fourwaymmr_test.py

```diff
@@ -175,7 +175,10 @@ def test_replicated_multivalued_entries(topo_m4):
     testuser.set("mail", ["test1", "test2", "test3"])
     # Now we check the entry on each consumer, making sure the order of the
     # multi-valued mail attribute is the same on all server instances
-    repl.wait_for_replication(topo_m4.ms["supplier4"], topo_m4.ms["supplier1"])
```

Comment:
If the robustness of the test was improved, should we remove the 'flaky' pytest.mark?

Reply:
Thanks @droideck for the review, yes you are correct. Let's remove the flaky mark for the moment.